### ISSUE TITLE
error

### ISSUE THREAD

**Issue by SSSLuckyStar** [GATECH-EIC/Edge-LLM#2]
Has anyone successfully run this project? I encountered a lot of errors during the process. What is the minimum GPU VRAM configuration required to run this project successfully?

**Comment by Jayzhou3219** [comment]
nope, this code lacks the necessary package version information and corresponding files.
